components/event-publisher-proxy/Makefile

```diff
@@ -66,3 +66,23 @@ $(cmds_images_push): %.push: %
 	@echo "+ Pushing container image $($@_img_name)"
 	docker image tag $($@_img_name) $(DOCKER_PUSH_REPOSITORY)$(DOCKER_PUSH_DIRECTORY)/$($@_img_name):$(TAG)
 	docker image push $(DOCKER_PUSH_REPOSITORY)$(DOCKER_PUSH_DIRECTORY)/$($@_img_name):$(TAG)
+
+test-local: ## Run all tests
+	mkdir -p /tmp/artifacts
+	go test -coverprofile=/tmp/artifacts/cover.out --tags="unit integration" ./...
+	@echo -n "Total coverage: "
+	@go tool cover -func=/tmp/artifacts/cover.out | grep total | awk '{print $$3}'
+ 
+test-local-unit: ## Run only unit tests
```

Comment:
Exec `make test-local-unit` failed:

```
pkg/handler/beb/handler_health_test.go:18:24: undefined: bigRequestSize
pkg/handler/beb/handler_health_test.go:19:24: undefined: defaultEventsEndpoint
```

Reply:
it makes actually sense, but I was not expecting this: if you define a const in a test file with the build tag `integration` you don't have access to it any more in a file in the same module but with the build tag `unit` if you run tests with the `unti` arg.

Reply:
on a second glance I decided that all the tests are integration tests anyhow